common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -1059,7 +1059,7 @@ def get_item(self, usage_key, depth=0, **kwargs):
                 log.debug("Found more than one item for '{}'".format(usage_key))
             return items[0]
 
-    def get_items(self, course_locator, settings=None, content=None, qualifiers=None, **kwargs):
+    def get_items(self, course_locator, settings=None, content=None, qualifiers=None, in_tree_only=False, **kwargs):
```

Comment:
@doctoryes Yes it's unfortunate but now at least it would have the capacity to only fetch items that are actually in course tree or are direct root items ( just like course, about or other detached categories). 

This would effectively help us to get rid of orphans.